mftf/2.3/suite.md

```diff
@@ -242,16 +255,18 @@ The element may contain [`<test>`], [`<group>`], and [`<module>`].
 A set of filters to be used for excluding tests from the suite.
```

Comment:
Same comment as above ... A set of filters that you can use to ....

Reply:
:ok: